RELEASE/scripts/CONSUME.ash

```diff
@@ -1299,6 +1299,11 @@ void main(string command)
 	{
 		buffer b;
 		b.append_diet(d);
+		string userForbidden = get_property("forbiddenStores");
+		if (!userForbidden.contains_text("3408540")) {
```

Comment:
technically this would also catch user IDs with an additional digit but with this as a substring within them, but... I think the odds of that being relevant are so low that it doesn't actually matter.